setup.py

```diff
@@ -19,7 +19,7 @@
 
 setuptools.setup(
     name = 'verticapy',  
-    version = '0.6.2',
+    version = '0.6.2-1',
```

Comment:
We decided that the version has 3 parts: MAJOR.MINOR.PATCH
(http://confluence.verticacorp.com:8080/display/DEV/VerticaPy+-+Python+library+for+Vertica+ml#VerticaPy-PythonlibraryforVerticaml-Release)
Please make it 0.6.3